docs/BRP-LO GBA mapping - Personen proxy.csv

```diff
@@ -0,0 +1,204 @@
+categorie;element;Persoon GBA variant;;;;;Persoon Proxy variant;;;;;aanpassing door proxy;afleiding of feature;Buitengemeentelijke autorisatie;Beperkt zoekresultaat;opmerking/issue;
```

Comment:
opmerkingen die ik bij gba variant heb gemaakt gelden hier ook:
- Kolomnaam "Persoon Proxy variant" is de "veldnaam"
- Volgorde anders: eerst de veldnaam, dan categorie/elementnummer, dan GbaPersoon veldnaam
- Waarde voor "Beperkt zoekresultaat" "Ja" invullen i.p.v. "P". Zelfde kan je dan ook doen bij "Buitengemeentelijke autorisatie"("Ja" i.p.v. "X")
- Volgorde van velden sorteren op categorie-groep-elementnummer
- nummer weglaten bij groepen
- opmerkingen/issue kolom kan vervallen. zijn allemaal te beantwoorden

Reply:
> * Kolomnaam "Persoon Proxy variant" is de "veldnaam"

Is gebeurd.

> * Volgorde anders: eerst de veldnaam, dan categorie/elementnummer, dan GbaPersoon veldnaam

Is gebeurd.

> * Waarde voor "Beperkt zoekresultaat" "Ja" invullen i.p.v. "P". Zelfde kan je dan ook doen bij "Buitengemeentelijke autorisatie"("Ja" i.p.v. "X")

Is gebeurd in beide csv bestanden.

> * Volgorde van velden sorteren op categorie-groep-elementnummer

Is gebeurd.

> * nummer weglaten bij groepen

Is gebeurd.

> * opmerkingen/issue kolom kan vervallen. zijn allemaal te beantwoorden

Is gebeurd in het csv bestand (nog niet in het Excel bestand.